### ISSUE TITLE
Page Properties force overwrites markdown with existing YAML Page Metadata

### ISSUE THREAD

**Issue by seafre** [Requarks/wiki#1925]
**Describe the bug**
Upon importing Markdown from existing source, the Page Properties blank fields force overwrite of the imported markdown's existing YAML Page Metadata. This breaks usage of wiki.js for managing existing source materials from Jekyll "Front Matter" etc.

**To Reproduce**

1. Import file
2. Edit file
3. Check git

_Pre-Import of "test-page.md":_
```
---
page: test-page
keywords: Test

lastupdated: "2019-01-18"
---
```

_Post-Import of "test-page.md":_
```
---
title: test-page
description: 
published: true
date: 2020-05-22T20:50:29.008Z
tags: 
---
```

**Expected behavior**
Existing keys are not over-written, and Wiki.js keys are optional (or prefixed with `wikijs-` ?

**Host Info (please complete the following information):**
 - OS: `Ubuntu Docker Container`
 - Wiki.js version: `2.2.51`
 - Database engine: `postgres 12.1`

**Comment by RyanGreenup** [comment]
This would also allow WikiJS to be compatible with [Notable](https://github.com/notable/notable) and so I'd love to see it.

---

This is quite related, I'm not sure if this should be it's own issue (if so I'll make a seperate issue), but if the `tags` property was modified to look like this

```
---
tags: [tag1, tag2, tag3]
---
# My MD Note
content blah...
```

That way it would:
1. Work with existing libraries
    * for example in **_R_** there is the `rmarkdown::yaml_front_matter()` function, this allows me to extract tags from markdown files, but without the square brackets it returns an error.
2. Work with [*Notable*](https://github.com/notable/notable) which also expects the square brackets.

Better Cross compatability would be a welcome improvement.

**Comment by seafre** [comment]
@regevbr Is this already covered by your recent PR for improving Page Properties?

**Comment by regevbr** [comment]
Nope

**Comment by seafre** [comment]
@NGPixel do you agree with the following proposal to resolve this issue?

**Metadata inject function:**
https://github.com/Requarks/wiki/blob/dev/server/helpers/page.js#L68
- Proposal: change `let` “meta”, with each array element using prefix `wikijs_`
- Proposal: replace `return` with for loop on array elements, if found existing metadata in page.content then replace - else append new line with the metadata

**Metadata read function calls:**
https://github.com/Requarks/wiki/blob/dev/server/modules/storage/disk/common.js#L89
- Proposal: change `_.get` to use prefixes such as wikijs_title, wikijs_description, wikijs_isPublished, wikijs_tags

----

If I've understood the inject function, it should have no impact on calls to Metadata inject function:
- https://github.com/Requarks/wiki/blob/dev/server/models/pages.js#L165
- https://github.com/Requarks/wiki/blob/dev/server/controllers/common.js#L89
- https://github.com/Requarks/wiki/blob/dev/server/modules/storage/git/storage.js#L426

If I've understood the read function, it should have no impact on calls to Metadata read function:
- https://github.com/Requarks/wiki/blob/dev/server/models/pages.js#L183

**Comment by NGPixel** [comment]
@seafre No, existing fields should be preserved but Wiki.js fields shouldn't be prefixed. Not only that would break any page already using un-prefixed fields, it wouldn't make sense to have duplicate fields per application.

**Comment by sean-freeman** [comment]
Too old to matter
